README.md

```diff
@@ -102,8 +102,10 @@ More Information  [llvm-cov](https://github.com/llvm-mirror/clang/blob/master/do
 ##### 3. Enable "Publish Coverage Report" publisher in the Post-build Actions.
 ##### 4. Add your coverage tool adapter and specify reports path.
 ![alt text](./images/config-add-adapter.png "Add coverage adapter")
-##### 5. (Optional) Specify Thresholds of each metrics in global or adapter level.
-##### 6. (Optional) Specify Source code storing level to enable source code navigation.
+##### 5. (Optional) Install the [forensics-api](https://github.com/jenkinsci/forensics-api-plugin) plugin 
```

Comment:
this is a mandatory dependency of the plugin? So remove optional?

Reply:
No if you do not use the forensics-api and find the reference branch, there wont be a delta coverage calculation but thats ok.

Reply:
This is a mandatory dependency:

![image](https://user-images.githubusercontent.com/21194782/119312362-2fd55f00-bc6a-11eb-916d-037ba7afab09.png)

That means it will be automatically installed for the user, and this step being worded wrong?

Reply:
The user has to trigger the reference search of the forensics api manually. If this is done, i am searching fir the reference build and calculate the diff. Otherwise the reference build wont be found and bo diff is calculated. I am not sure if if get your point?

Reply:
The bit I am objecting to is '(Optional) Install the [forensics-api](https://github.com/jenkinsci/forensics-api-plugin) plugin'

The user cannot do this as it is force installed for them as it's a mandatory dependency

Reply:
Oh yes, of course.. will change this, thanks!